libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -782,6 +784,7 @@ public void logout(Activity frontActivity, final boolean showLoginPage) {
      * @param showLoginPage If true, displays the login page after removing the account.
      */
     public void logout(Account account, Activity frontActivity, final boolean showLoginPage) {
+        EventBuilderHelper.createAndStoreEvent("userLogout", null, TAG, null);
```

Comment:
Yes - that's why the call is before `logout`. The missing piece here (coming in another PR) is a fire-and-forget `publish` call just before `logout`.